lib/bourne-shell.sh

```diff
@@ -3,7 +3,13 @@
 # see /usr/share/doc/bash/examples/startup-files (in the package bash-doc)
 
 # make less more friendly for non-text input files, see lesspipe(1)
-[ -x /usr/bin/lesspipe ] && eval "$(SHELL=/bin/sh lesspipe)"
+if [ -z "${LESSOPEN}" ] && [ ! -f /etc/gentoo-release ]; then
```

Comment:
You can optionally add a code for Gentoo, e.g. `export LESSOPEN='| /usr/bin/lesspipe %s'`, because we know the behavior of `/usr/bin/lesspipe` in Gentoo systems and is not supposed to be changed by users. <!-- But, if you want to address the case for Gentoo after the Gentoo maintainer for `less` has responded, this is OK for now. -->